src/components/Register/Register.js

```diff
@@ -47,42 +47,37 @@ class Register extends React.Component {
   }
 
   handleSubmitForm = (e) => {
-    this.handleCheckUsername();
+    const { registerStart } = this.props;
+    registerStart();
     e.preventDefault();
   }
 
-  handleCheckUsername = () => {
-    const { username, checkExist } = this.props;
-    if (username !== '') {
-      checkExist(username);
-      this.setState({
-        validate: true
-      })
-    }
+  componentDidUpdate = (prevProps, prevState) => {
+    const { username, registered, isRegistering, register, registerSuccess, registerFailed, exist, loadAccount } = this.props;
```

Comment:
Lần sau nếu lấy nhiều hơn 4 biến trên 1 hàng thì tách xuống hàng ntn nha

```suggestion
const {
username, registered, isRegistering,
register, registerSuccess, registerFailed,
exist, loadAccount,
} = this.prop
```